doc/source/internals.rst

```diff
@@ -15,7 +15,8 @@
 Internals
 *********
 
```

Comment:
maybe we remove internals entirely

Reply:
I think there's still some useful information there about how pandas is structured, but that's too technical for `contributing.rst`.